sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Rules/TestMethodShouldHaveCorrectSignature.cs

```diff
@@ -63,5 +63,28 @@ public void TestMethodShouldHaveCorrectSignature_Xunit(string testFwkVersion)
                     .Concat(MetadataReferenceHelper.FromNuGet("xunit.extensibility.core", testFwkVersion))
                     .ToArray());
         }
+
+        [TestMethod]
+        [TestCategory("Rule")]
+        public void TestMethodShouldHaveCorrectSignature_Xunit_Legacy()
```

Comment:
I haven't tested but maybe we could still have a `DataTestMethod` testing with version `1.7.0.1540` (older) and `1.9.1`.

Also that's surprising the version `1.9.2` doesn't also have this behavior.

Reply:
I don't understand - what behaviour? Xunit doesn't have _DataTestMethod_.
The main purpose of this test is to check that we recognize the legacy XUnit attributes. I don't think there's any value in checking more than one old version, unless we know that there are multiple old versions of the API that we need to support.